recipe/meta.yaml

```
@@ -37,16 +38,21 @@ test:
     - nose2.tests.functional
     - nose2.tests.unit
     - nose2.tools
+  requires:
+    - pip
   commands:
+    - pip check
     - nose2 --help
-    - nose2-3.6 --help
-
+    - nose2-{{ python }} --help
 about:
   home: https://github.com/nose-devs/nose2
+  description: The successor to nose, based on unittest2
   summary: nose2 is the next generation of nicer testing for Python
-  license: BSD
+  license: BSD-3-Clause
   license_family: BSD
   license_file: license.txt
+  dev_url: https://docs.nose2.io/en/latest/
```

This is a more approprivate dev_url: https://docs.nose2.io/en/latest/

---

ah, my bad, you mean this:

```suggestion
  dev_url: https://github.com/nose-devs/nose2
```

---

Yes, that's exactly what I meant - thanks for reading my mind 😆